kyuubi-server/src/main/scala/org/apache/kyuubi/server/statestore/api/Metadata.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server.statestore.api
+
+/**
+ * The metadata to store. It including three parts:
+ * 1. session related metadata.
+ * 2. request related metadata.
+ * 3. engine related metadata.
+ *
+ * @param identifier the identifier id.
+ * @param username the user name.
```

Comment:
miss docs of `realUser` and the docs of `username` is wrong, it should mention to the proxy user.